compiler/src/iree/compiler/Modules/HAL/Loader/Transforms/Conversion.cpp

```diff
@@ -49,11 +50,19 @@ class ConversionPass : public ConversionBase<ConversionPass> {
         mlir::func::FuncDialect, mlir::scf::SCFDialect,
         mlir::arith::ArithDialect, mlir::affine::AffineDialect>();
 
-    TypeConverter typeConverter;
+    SmallVector<const HALConversionDialectInterface *> conversionInterfaces;
```

Comment:
please make the same change in HAL/Inline/Transforms/Conversion.cpp

Reply:
Implemented.